opba-facade-protocol-api-shared/pom.xml

```diff
@@ -26,5 +26,11 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Please do not add entire dependency to Facade (that is going to be shared across all protocols) just for one class. Simply recreate `ChallengeData` class within our project

Reply:
Dependency removed